macros/anaUltraThreshold.py

```diff
@@ -244,6 +244,7 @@
 r.gStyle.SetOptStat(0)
 print 'Saving vt1Max distributions'
 for vfat in range(0,24):
+    if (vfat_mask >> vfat) & 0x1: continue
     r.gStyle.SetOptStat(0)
     canv_vt1Max.cd(vfat+1)
     dict_hMaxVT1[vfat].Draw("hist")
```

Comment:
for vfat in dict_hMaxVT1.keys():
that only needs to be
for vfat in dict_hMaxVT1:
since python does these same thing in both cases. using a for in on a dict iterates over the keys.